com.ibm.wala.dalvik/build.gradle

```diff
@@ -138,7 +138,7 @@ dependencies {
 			'junit:junit:4.13',
 			'org.osgi:org.osgi.core:6.0.0',
 			'org.smali:dexlib2:2.4.0',
-			files(installAndroidSdk.map { "${it.component('build-tools')}/lib/dx.jar" }),
+			'com.android.tools:r8:1.6.84',
```

Comment:
Please move “`'com.android.tools:r8:1.6.84',`” to just before the “`'junit:junit:4.13',`” line. We keep dependencies in sorted order, in part to emphasize that their order does not matter, and in part to make duplicates easier to spot.